circle.yml

```diff
@@ -0,0 +1,15 @@
+# circle.yml
+machine:
+  services:
+    - docker
+  environment:
+    EXPOSED_PORT: 8080
+
+dependencies:
+  override:
+    - pip install docker-compose
+    - EXPOSED_PORT=8080 docker-compose build
+
+test:
+  override:
+    - docker-compose run --rm web pytest
```

Comment:
Ah that makes sense of what's going on then.  I'll happily pair with one of you to figure out at least a smoketesting strategy for yarnitor but I am not familiar enough with web tech to figure it out in any sort of reasonable time on my own